lib/setup.js

```diff
@@ -58,7 +56,7 @@ function setupTestServer (opt, cb) {
       return function (req, res) {
         var args = [].slice.call(arguments)
         if (isControlRequest(req)) {
-          args.push({ target: localUrl(controlPort) })
+          args.push({ target: localUrl(opts.control_port) })
```

Comment:
Following what `standard` enforces, shouldn't that be camelCase?

Reply:
Not on object properties.

Reply:
Ah, interesting.

Reply:
Personally I don't like it, but I just refactored the code.

Reply:
I'm sure @feross can weigh in more. I think enforcing it on object properties would be problematic, if not stupid :)